lib/amazon_ssa_support/ssa_queue.rb

```diff
@@ -159,7 +159,8 @@ def get_reply(msg)
         s3_obj = @reply_bucket.object(s3_obj_name)
         unless s3_obj.exists?
           msg.delete
-          raise "Reply object #{s3_obj_name} does not exist"
+          $log.error("Reply object #{s3_obj_name} does not exist")
```

Comment:
Going forward, we should also flag the job so the error will be displayed on the task page.
I do understand that the Job id is not available when we don't have the `s3_obj`, so the fix for this is not a simple one.
Given this is an edge case, I think this is OK for now, but we should look into addressing it in the future.

Reply:
Yeah actually if you can hold off on merging this - I may be able to pull out the job_id right here and return it.  Testing now...